src/pages/sns.tsx

```diff
@@ -0,0 +1,480 @@
+                social media platforms, B2B enterprise services, play-to-earn
+                games and online news media outlets can now truly become
+                decentralized.
+              </motion.p>
+              <motion.p
+                className="mb-6 tw-paragraph"
+                variants={transitions.item}
+              >
+                All parts of such applications are owned and governed by an SNS
+                DAO. This is in contrasts to existing DAOs whereby governance
+                applies only to smart contract logic, which is just a small part
+                of any web3 application.{" "}
+                <span className="italic">No single entity owns a dapp.</span>
+              </motion.p>
+              <MotionLink
+                className="link-external"
+                href="https://internetcomputer.org/docs/current/tokenomics/sns/sns-intro-tokens"
+                variants={transitions.item}
+              >
+                Learn more
+              </MotionLink>
+            </Card>
+            <Card
+              title="Autonomous"
+              icon={<IconAutonomous aria-hidden="true" />}
+            >
+              <motion.p
+                className="mb-6 tw-paragraph"
+                variants={transitions.item}
+              >
+                Liquid democracy. When a proposal is approved by the community,
+                the SNS implements changes to the code and upgrades itself
+                autonomously. This ensures permisionless and decentralized
+                governance of an SNS DAO.{" "}
+                <span className="italic">
+                  No single entity can stop the process.
+                </span>
+              </motion.p>
+              <MotionLink
+                className="link-external"
+                href=""
+                variants={transitions.item}
+              >
+                Learn more
+              </MotionLink>
+            </Card>
+            <Card title="Tokenized" icon={<IconTokenized aria-hidden="true" />}>
+              <motion.p
+                className="mb-6 tw-paragraph"
+                variants={transitions.item}
+              >
+                Financially independent. Each SNS DAO has its own utility token
+                that community members can stake to claim governance rights or
+                use to purchase services. The same token can be used to reward
+                community contributions to the SNS DAO. Such contributions can
+                range from engaging with the dapp to writing code, designing,
+                promoting or participating in governance.{" "}
+                <span className="italic">No fiat required.</span>
+              </motion.p>
+              <MotionLink
+                className="link-external"
+                href="https://internetcomputer.org/docs/current/tokenomics/sns/sns-intro-tokens"
```

Comment:
Should this link to https://internetcomputer.org/docs/current/tokenomics/sns/tokenomics ? 
If I press on the current link, I end up on the top level "SNS" page..

Reply:
Removing this link for now until all docs are live.